test_harness/src/lib.rs

```diff
@@ -332,21 +330,17 @@ mod tests {
                     IMPRESSION,
                     Pricing {
                         // 0.04000000
-                        // Per 1000 = 40.00000000
                         min: 4_000_000.into(),
```

Comment:
FromWhole is perfect in this situation, you might change it but it's not required.